packages/material-ui-styles/src/makeStyles/makeStyles.test.js

```diff
@@ -116,11 +116,32 @@ describe('makeStyles', () => {
       const mountWithProps2 = createGetClasses(styles);
       expect(() => {
         mountWithProps2({});
-      }).to.throw();
+      }).to.throw('theme.spacing is not a function');
       expect(consoleErrorMock.callCount()).to.equal(4);
+      expect(consoleErrorMock.messages()[0]).to.include(
+        'Material-UI: The `styles` argument provided is invalid.\nYou are providing a function without a theme in the context.',
```

Comment:
This error message is assumes we're using the package in core with components relying on the theme. The error caught does not necessarily have to be caused by a missing theme e.g. `() => { throw new Error() }` would trigger this warning even though it's unrelated to a missing theme.

Because this package will go away soon I'm not going to change the error message. Especially without any reports that I'm aware of that find this warning confusing (but confusing error messages are probably underreported anyway)